app/code/community/TIG/MyParcel2014/Model/Api/MyParcel.php

```diff
@@ -805,6 +807,31 @@ protected function _getConsignmentData(TIG_MyParcel2014_Model_Shipment $myParcel
             $data['physical_properties']['weight'] = (int)$totalWeight;
         }
 
+        if($data['options']['package_type'] == self::DIGITAL_STAMP){
+            $totalWeight = 0;
+            $items = $myParcelShipment->getOrder()->getAllItems();
+            foreach($items as $item) {
```

Comment:
$totalWeight is hierboven al bekend. Die kan je gebruiken.

Reply:
Done!